vm-setup/roles/v1aX_integration_test/tasks/upgrade.yml

```
@@ -147,26 +156,48 @@
       state: directory
       recurse: yes
     with_items:
+    - dev-repository/cluster-api/{{ CAPIRELEASE }}
+    - dev-repository/bootstrap-kubeadm/{{ CAPIRELEASE }}
+    - dev-repository/control-plane-kubeadm/{{ CAPIRELEASE }}
+    - overrides/infrastructure-metal3/{{ CAPIRELEASE }}
     - dev-repository/cluster-api/{{ CAPI_REL_TO_VERSION }}
     - dev-repository/bootstrap-kubeadm/{{ CAPI_REL_TO_VERSION }}
     - dev-repository/control-plane-kubeadm/{{ CAPI_REL_TO_VERSION }}
     - overrides/infrastructure-metal3/{{ CAPM3_REL_TO_VERSION }}
 
+  - name: Copy files
+    vars:
+      working_dir: "{{HOME}}/.cluster-api"
+    copy: src="{{working_dir}}/{{item.src}}" dest="{{working_dir}}/{{item.dest}}"
+    with_items:
+    - {
+        src: "dev-repository/cluster-api/v0.4.99/core-components.yaml",
+        dest: "dev-repository/cluster-api/{{CAPIRELEASE}}/core-components.yaml"
+      }
```

```suggestion
    - {
        src: "dev-repository/cluster-api/{{CAPIRELEASE_HARDCODED}}/core-components.yaml",
        dest: "dev-repository/cluster-api/{{CAPIRELEASE}}/core-components.yaml"
      }
```
The same change on the other lines as well.